website/projects/awssync.py

```diff
@@ -22,8 +26,37 @@ def button_pressed(self):
         :return: True if function executes successfully
         """
         self.logger.info("Pressed button")
+        self.logger.info(self.get_emails_with_teamids())
         return True
 
+    def get_all_mailing_lists(self):
+        """
+        Get all mailing lists from the database.
+
+        :return: List of mailing lists
+        """
+        mailing_lists = MailingList.objects.all()
+        mailing_list_names = [ml.email_address for ml in mailing_lists]
+        return mailing_list_names
+
+    def get_emails_with_teamids(self):
+        """
+        Create a tuple with email and corresponding teamID, where teamID is a concatenation of ID and semesterID.
+
+        :return: list of (email, teamid)
+        """
+        mailing_lists = MailingList.objects.all()
+        email_id = []
+        for ml in mailing_lists:
+            project = ml.projects.all()
+            if len(project) != 0:
+                project_id = [(p.id, p.semester.id) for p in project]
+                project_id = int(str(project_id[0][0]) + str(project_id[0][1]))
```

Comment:
Notice that the inner Meta class of the Project class has the attribute `unique_together` for uniqueness constraints. You should perhaps use these for unique string identifiers per project. Is there a reason why it must strictly be of type integer like you have it now?